src/layout/List/ListComponent.tsx

```diff
@@ -53,6 +54,11 @@ export const ListComponent = ({
     }
   };
 
+  const tableHeadersValues = {...tableHeaders};
```

Comment:
Fixing linting issue
```suggestion
  const tableHeadersValues = { ...tableHeaders };
```